platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/DefaultBuildTreeModelControllerServices.kt

```diff
@@ -181,7 +181,7 @@ class DefaultBuildTreeModelControllerServices : BuildTreeModelControllerServices
                     configureOnDemand = configureOnDemand,
                     configurationCache = configurationCache,
                     isolatedProjects = isolatedProjects,
-                    intermediateModelCache = false,
+                    intermediateModelCache = isolatedProjects,
```

Comment:
`isolatedProjects` is now always == `intermediateModelCache`. Should we just remove `intermediateModelCache` and replace its usages with `isolatedProjects`?